core/src/main/java/org/jdbi/v3/core/statement/SqlStatement.java

```diff
@@ -1217,6 +1217,66 @@ public final This bindList(String key, List<?> values) {
     }
 
     /**
+     * Bind a parameter for each value in the given list * number of property names,
+     * and defines an attribute as the comma-separated list of parameter references (using colon prefix).
+     *
+     * Used to create query similar to:
+     * select * from things where (id, foo) in ((1,'abc'),(2,'def'),(3,'ghi'))
+     * <p>
+     * Examples:
+     * <pre>
+     *
+     * List&lt;Things&gt; thingsToSearchBy = ...
+     * List&lt;Thing&gt; things = handle.createQuery("select * from things where (id, foo) in (&lt;thingKeys&gt;)")
```

Comment:
Oh, this might be tricky if your column naming strategy is anything other than directly the same as the Java bean property name.  Maybe this won't work.